charts/memgraph-high-availability/README.md

```diff
@@ -0,0 +1,72 @@
+## Memgraph high availability Kubernetes Helm Chart
+A Helm Chart for deploying Memgraph in [high-availability setup](https://memgraph.com/docs/clustering/high-availability).
```

Comment:
```suggestion
A Helm Chart for deploying Memgraph in [high availability setup](https://memgraph.com/docs/clustering/high-availability).
```

TBH not sure, but we've been using - for highly-available, not for high availability